conf/messages.cy

```diff
@@ -210,5 +211,6 @@ charityUKAddress.officialAddress.outsideUK.link = Charity’s official address i
 # CharityContactAddressChoicePage Messages
 # ----------------------------------------------------------
 isCharityOfficialAddressInUK.title = Is your charity''s official address in the UK?
+isCharityOfficialAddressInUK.section = Charity Information
```

Comment:
Section name 'Charity Information' is same for all pages in Section1... so it would be better to have one message key like 
contactDetail.Section = Charity Information
and delete other section messages

Reply:
Done as per comment